src/creator.rs

```diff
@@ -55,7 +55,7 @@ impl<H: Hasher> Creator<H> {
 
     // initializes the vectors corresponding to the given round (and all between if not there)
     fn init_round(&mut self, round: Round) {
-        if (round + 1) as usize > self.n_candidates_by_round.len() {
+        if usize::from(round + 1) > self.n_candidates_by_round.len() {
```

Comment:
As far as I know currently old way is consistent with rest of the repo. Did you already change this in other files?

Reply:
I don't see any point in fixing it everywhere else in this PR. It's just a small local improvement. I'm not attached to it. Just changed it since quite often I'm asked to do similar improvements when somebody notice something worth to be bothered.

Reply:
I am more concerned about it's intended behavior. Is it safe to call `self.create_unit()` if this method exits before reaching its intended state?